### PR TITLE
load_dumps.php added

### PR DESCRIPTION
**Pull request by chandan5**
To load just run php load_dumps.php. Sql files are in databases folder.
Check if it gets loaded at yours pc. It gets loaded on mine.

### CODE REVIEW

**Comment by gauravmittal1995** [comment]
Um, right now our plan is to only load the database which is required for the disaster that the deployment is for. This loads all the data. Once the Deployment Settings gets completed, we can merge this using them. But Thanks for this PR. This will make it easy to setup the db.

**Comment by chandan5** [comment]
:+1:

**Comment by gauravmittal1995** [comment]
Also, @nisargjhaveri and @chandan5  , i was thinking that we load and run the script the first time our app runs. i.e if the db is not prepopulated, we can run this script. This will make it simpler for the user to avoid running any external script and keep the deployment process self contained. Thoughts?

File: load_dumps.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+require('app/config.php');
```
Comment (gauravmittal1995) [inline load_dumps.php:2]: Also i think you can avoid this line as the framework automatically includes the config.php file in all the php scripts. But check it once anyways

File: load_dumps.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+$db = new PDO('mysql:dbname='.DB_NAME, DB_USER, DB_PASS);
+
+$sql = file_get_contents('databases/saathi.sql');
+$qr = $db->exec($sql);
```
Comment (gauravmittal1995) [inline load_dumps.php]: Why are you assigning a variable if its not being used. I think you can check for the success and display any error.

<pre>
if ($qr) {
    // Report Success
} else {
    // Report Failure
}
</pre>

File: load_dumps.php (hunk 1 above)
Comment (chandan5) [inline load_dumps.php:2]: This script was supposed to run separately on command line. Currently it is not called from the context of framework.

Comment (gauravmittal1995) [inline load_dumps.php:2]: I see. Very well then :+1:

File: load_dumps.php (hunk 2 above)
Comment (chandan5) [inline load_dumps.php]: Script does not give any error even if same insert operation is run again.
So error checking is not done. Removed extra variable name $qr.

**Comment by chandan5** [comment]
Yeah its a good idea to load and run the script first time our app runs. +1 for that

**Comment by nisargjhaveri** [comment]
I would prefer a `setup` script that setups the DB and also create the content of `config.php` with all the config options including DB connection set properly based on input from user. Has anyone seen setup of 'WordPress'?

**Comment by gauravmittal1995** [comment]
@nisargjhaveri +1 Thats probably the best idea. Yes, the wordpress 5 minute setup is really known for its ease. We can do something like that.

**Comment by chandan5** [comment]
@nisargjhaveri +1 We can do that and it will be good.

**Comment by nisargjhaveri** [comment]
See #92 for further developments.

**Pull request closed without merging**